float8_experimental/float8_linear_utils.py

```diff
@@ -70,8 +70,10 @@ def _update_history_with_new_amax(new_amax, amax_history):
 def swap_linear_with_float8_linear(
     module: nn.Module,
     module_cls: Type[nn.Module],
-    emulate: bool = False,
+    *,
```

Comment:
I was thinking that forcing these to be kwarg only would be less error prone.

Reply:
yeah I like it!